app/views/layouts/application.html.erb

```diff
@@ -69,6 +69,14 @@
       </div>
       <nav class="footer">
         <div class="container">
+          <div class="col-md-4">
+            <div class="row">
+              <%= label_tag :search_issue, t(:edit_issue), class: 'col-md-6 col-form-label' %>
+              <div class="col-md-6">
+                <%= text_field_tag :search_issue, nil, class: 'form-control', placeholder: 'Nummer', maxlength: 6 %>
```

Comment:
Den Placeholdertext bitte via I18n holen, (`placeholder.number`)

Generell sollen möglichst keine Texte aus den Tempplates direkt kommen , damit man über ein einfaches Overlay in der Locale die Texte beeinflussen kann.